src/LibExpat.jl

```diff
@@ -357,7 +357,7 @@ function find{T<:String}(pd::ETree, path::T)
             end
 
             if m.captures[3] != nothing
-                push!(xp, (:filter, (:number,int(m.captures[3]))))
+                push!(xp, (:filter, (:number,Int(m.captures[3]))))
                 idx = true
```

Comment:
Needs to be `Base.parse(Int, m.captures[3])`

(the `Base` qualification is necessary because the `parse` in this package overwrites `Base.parse`, which is now a heavily used function in Base -- if the Base function stays that way, this package's `parse` will need to be renamed, or unexported (ref'd as `LibExpat.parse`) in the long term -- no need to do this now as things are in flux in Base.)